BossMod/Framework/Plugin.cs

```diff
@@ -81,6 +82,9 @@ public unsafe Plugin(IDalamudPluginInterface dalamud, ICommandManager commandMan
         _ai = new(_rotation, _amex);
         _broadcast = new();
         _ipc = new(_rotation, _amex);
+        _dtr = new(_rotation);
+
+        Service.Framework.Update += OnUpdate;
```

Comment:
Why do you need it to happen on framework update, rather than on usual plugin update (which happens during draw)?

Reply:
oh no idea I copied this from vnavmesh, i can move it

Reply:
When you update it, pls merge to master, i've recently merged other PR which added some dtr stuff, I think it's duplicated now.